examples/slims_launcher.py

```diff
@@ -0,0 +1,125 @@
+import logging
+import subprocess
+from pathlib import Path
+from typing import Literal, Optional, Self
+
+from aind_behavior_services.rig import AindBehaviorRigModel
+from aind_behavior_services.session import AindBehaviorSessionModel
+from aind_behavior_services.task_logic import AindBehaviorTaskLogicModel
+from pydantic import Field
+from pydantic_settings import CliApp
+from typing_extensions import override
+
+from aind_behavior_experiment_launcher import resource_monitor
+from aind_behavior_experiment_launcher.apps import App
+from aind_behavior_experiment_launcher.behavior_launcher import (
+    BehaviorCliArgs,
+    BehaviorLauncher,
+    BehaviorServicesFactoryManager,
+    SlimsPicker,
+)
+
+logger = logging.getLogger(__name__)
+
+TASK_NAME = "RandomTask"
+LIB_CONFIG = rf"local\AindBehavior.db\{TASK_NAME}"
+
```

Comment:
This is not being used?

Reply:
Can you just add a TODO here to remind us to refactor this someday to use a common module?